source/_patterns/04-components/menus/menu--main/_menu--main.scss

```diff
@@ -3,4 +3,94 @@
 
 .menu--main {
   @include list-inline;
+
+  .menu__link.has-subnav {
+    position: relative;
+
+    &::after {
+      color: gesso-grayscale(white);
+      content: '\02192';
+      display: block;
+      height: 10px;
+      opacity: 1;
+      position: absolute;
+      right: 5px;
+      top: 50%;
+      transform: translateY(-50%);
+      width: 10px;
+    }
+  }
+
+  > .menu__item {
+    display: inline-block;
+    margin-right: rem(gesso-spacing(lg));
+    position: relative;
+
+    > .menu__link {
+      @include link(gesso-grayscale(gray-4), gesso-palette(blue, base));
```

Comment:
we shouldn't use any palette colors in the default components.  Instead use colors from the 'colors' section...  maybe something from UI section or create a new one for menus?